Orange/widgets/visualize/tests/test_owboxplot.py

```diff
@@ -281,8 +280,8 @@ def _select_data(self):
                  if isinstance(item, FilterGraphicsRectItem)]
         items[0].setSelected(True)
         return [100, 103, 104, 108, 110, 111, 112, 115, 116,
-                120, 123, 124, 126, 128, 132, 133, 136, 137,
-                139, 140, 141, 143, 144, 145, 146, 147, 148]
+                120, 123, 124, 128, 132, 133, 136, 137, 139,
+                140, 141, 143, 144, 145, 146, 147]
```

Comment:
A slight change because the `np.nanquartiles` does not put the threshold at the same place as `_quantiles`.